.github/workflows/optimize-images.yml

```diff
@@ -0,0 +1,61 @@
+# Release builds that are triggered by the v*.*.* tag. Release tags occur when
+# someone comments /approve on the associated release ticket
+
+name: '[Build] Release Build'
+
+on:
+  push:
+    branches:
+      - 'release/v[0-9]+.[0-9]+.[0-9]+'
+      - 'chanel-6809-add--mage-optimization-into-ci-for-app-store-submissions'
+
+jobs:
+
+  optimize_images:
+    name: Optimize Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install imagemin-cli
+        run: npm install -g imagemin-cli
+      - name: List images before optimization
+        run: |
+          echo "iOS images before optimization:"
+          ls -lh VAMobile/ios/fastlane/screenshots/en-US
+          echo "Android images before optimization:"
+          ls -lh VAMobile/android/fastlane/metadata/android/en-US/images/
+          ls -lh VAMobile/android/fastlane/metadata/android/en-US/images/phoneScreenshots/
+          ls -lh VAMobile/android/fastlane/metadata/android/en-US/images/sevenInchScreenshots/
```

Comment:
Are these list image steps before and after necessary? Maybe something more useful would be to list the # of images that were optimized afterwards. Could potentially count the # of files modified below on line 58.

Reply:
I added the list before and after for debug purposes. If it can count the number of opmitimized images, that will be great!!